test_scripts/WebEngine/WebEngineProjection/001_Registration.lua

```diff
@@ -23,6 +23,7 @@
 local common = require('test_scripts/WebEngine/commonWebEngine')
 
```

Comment:
```suggestion
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-4f3559029bee4e49c37fb96dd1163cc6R26)